schemas/group.yml

```diff
@@ -26,3 +26,11 @@
       type: keyword
       description: >
         Name of the group.
+
+    - name: domain
+      level: extended
+      type: keyword
+      description: >
```

Comment:
Please add a `short` description as well, with just the first sentence.

`short: Name of the directory the group is a member of.`

Don't forget to run make prior to committing ;-)

Reply:
Done! Pls let me know if everything is OK